.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,6 +6,8 @@
 
 ## Checklist
 
+- Is this targeting a **general** release / a **preview** release / an **engineering build** of Azure PowerShell?
+    - `Private Preview`
```

Comment:
Could you explain more about the Private Preview case? 
- Why it is listed singularly?
- Where should the pr go if it is a private preview?

Reply:
To explain the context, I updated "CONTRIBUTING.md" and link it here.